{
    "functionName": "test_ge_incompatible_units",
    "className": "TestValidate_geq",
    "fileName": "/kyleniemeyer_&_PyTeCK/pyteck_&_tests_&_test_validation.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "with pytest.raises(DimensionalityError):\n    validate_geq('testval', 5 * units.second, 1 * units.meter)\n"
}